code/modules/mob/living/basic/space_fauna/spider/spiderlings/spiderling.dm

```diff
@@ -10,20 +10,22 @@
 	desc = "It never stays still for long."
 	icon_state = "spiderling"
 	icon_dead = "spiderling_dead"
+	density = FALSE
```

Comment:
If you did this in a separate PR already then doing it here too means you will conflict with yourself when it is merged.